src/components/Form/Fields.js

```diff
@@ -28,39 +28,41 @@ export const { TextField, Checkboxes } = Fields;
 
 export const Select = (props) => {
     const { select } = useSelectStyles();
-    return <Fields.Select className={select} {...props} />;
+    return <Fields.Select style={{marginTop: "16px", marginBottom: "24px"}} className={select} {...props} />;
```

Comment:
if you're applying all the same styling for all of these fields, make this a jss class instead